File: Find the Highest Altitude/find-the-highest-altitude.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        s = 0
        maxs = 0
        for g in gain:
            s += g
            maxs = max(maxs, s)
        return maxs

File: Find the Highest Altitude/find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int s=0;
        int maxs=0;
        for(int i=0;i<gain.size();i++){
            s+=gain[i];
            maxs=max(maxs,s);
        }
        return maxs;
    }
};